collect_app/src/main/java/org/odk/collect/android/draw/PenColorPickerDialog.kt

```diff
@@ -0,0 +1,39 @@
+package org.odk.collect.android.draw
+
+import android.app.Dialog
+import android.content.Context
+import android.os.Bundle
+import androidx.fragment.app.DialogFragment
+import androidx.lifecycle.ViewModelProvider
+import com.google.android.material.dialog.MaterialAlertDialogBuilder
+import com.rarepebble.colorpicker.ColorPickerView
+import org.odk.collect.android.injection.DaggerUtils
+import org.odk.collect.androidshared.R
+import javax.inject.Inject
+
+class PenColorPickerDialog : DialogFragment() {
+    @Inject
+    lateinit var factory: PenColorPickerViewModel.Factory
+
+    private lateinit var model: PenColorPickerViewModel
```

Comment:
Just discovered that there is a less noisy way to set up `ViewModels` that I think you'll like:

```kotlin
@Inject
lateinit var factory: PenColorPickerViewModel.Factory
private val model: PenColorPickerViewModel by activityViewModels { factory }
```

Reply:
Done.